TombLib/TombLib/Catalogs/Engines/TR1/Moveables.xml

```diff
@@ -118,9 +118,11 @@
     <moveable id="138" name="Lock 2" essential="false" ten="KEY_HOLE2" />
     <moveable id="139" name="Lock 3" essential="false" ten="KEY_HOLE3" />
     <moveable id="140" name="Lock 4" essential="false" ten="KEY_HOLE4" />
-    <moveable id="145" name="Complete Scion" id2="141" ten="KEY_ITEM1" />
-    <moveable id="146" name="Complete Scion" id2="142" ten="KEY_ITEM2" />
```

Comment:
`id2` field is used in TE to convert visible 3D items in editor back to sprite pickup objects in TR1 and TR2.

Reply:
Yes, but these are wrong. These objects are the animated scions  (Atlantis and Great Pyramid in OG) so they're not pickup objects. To add a normal scion pickup, you would add object 150.
This problem is mentioned in detail in the issue.

Reply:
Sorry, I remember reading about it somewhere :)